[MySQL 8.0] Init Container issue

I am trying to deploy a `MySQL 8.0` cluster using the `MySQL Operator`. However, I am getting these errors in the logs of the Mysql container:
```
kubectl logs -f mysql-cluster-test-mysql-0 mysql -n test

mysqld: Table 'mysql.plugin' doesn't exist
2019-07-03T18:16:51.902943Z 0 [Warning] [MY-011068] [Server] The syntax 'expire-logs-days' is deprecated and will be removed in a future release. Please use binlog_expire_logs_seconds instead.
2019-07-03T18:16:51.904587Z 0 [System] [MY-010116] [Server] /usr/sbin/mysqld (mysqld 8.0.15-6) starting as process 1
2019-07-03T18:16:52.461051Z 1 [Warning] [MY-010005] [Server] Skip re-populating collations and character sets tables in read-only mode.
2019-07-03T18:16:52.463437Z 0 [ERROR] [MY-010735] [Server] Can't open the mysql.plugin table. Please run mysql_upgrade to create it.
2019-07-03T18:16:52.562848Z 0 [Warning] [MY-010015] [Repl] Gtid table is not ready to be used. Table 'mysql.gtid_executed' cannot be opened.
2019-07-03T18:16:52.565228Z 0 [Warning] [MY-010068] [Server] CA certificate ca.pem is self signed.
2019-07-03T18:16:52.567938Z 0 [Warning] [MY-010441] [Server] Failed to open optimizer cost constant tables
2019-07-03T18:16:52.568079Z 0 [ERROR] [MY-013129] [Server] A message intended for a client cannot be sent there as no client-session is attached. Therefore, we're sending the information to the error-log instead: MY-001146 - Table 'mysql.component' doesn't exist
2019-07-03T18:16:52.568092Z 0 [Warning] [MY-013129] [Server] A message intended for a client cannot be sent there as no client-session is attached. Therefore, we're sending the information to the error-log instead: MY-003543 - The mysql.component table is missing or has an incorrect definition.
2019-07-03T18:16:52.568181Z 0 [ERROR] [MY-000067] [Server] unknown variable 'query-cache-size=0'.
2019-07-03T18:16:52.568190Z 0 [Warning] [MY-010952] [Server] The privilege system failed to initialize correctly. If you have upgraded your server, make sure you're executing mysql_upgrade to correct the issue.
2019-07-03T18:16:52.568549Z 0 [ERROR] [MY-010119] [Server] Aborting
2019-07-03T18:16:54.635552Z 0 [System] [MY-010910] [Server] /usr/sbin/mysqld: Shutdown complete (mysqld 8.0.15-6)  Percona Server (GPL), Release 6, Revision 63abd08.
```

The version of `MySQL` is set to `8.0.15`. 

Thanks

Some probably-related error messages in `MySQL` container of pod 1:
```
2019-07-04T18:07:07.992788Z 1 [Warning] [MY-012351] [InnoDB] Tablespace 2, name 'sys_operator/status', file './sys_operator/status.ibd' is missing!
2019-07-04T18:07:08.563943Z 6 [ERROR] [MY-012592] [InnoDB] Operating system error number 2 in a file operation.
2019-07-04T18:07:08.563966Z 6 [ERROR] [MY-012593] [InnoDB] The error means the system cannot find the path specified.
2019-07-04T18:07:08.563974Z 6 [ERROR] [MY-012216] [InnoDB] Cannot open datafile for read-only: './sys_operator/status.ibd' OS error: 71
2019-07-04T18:07:08.564165Z 6 [ERROR] [MY-000061] [Server] 1812  Tablespace is missing for table `sys_operator`.`status`..
```

The full logs:
```
2019-07-04T18:07:07.248070Z 0 [Warning] [MY-011068] [Server] The syntax 'expire-logs-days' is deprecated and will be removed in a future release. Please use binlog_expire_logs_seconds instead.
2019-07-04T18:07:07.249919Z 0 [System] [MY-010116] [Server] /usr/sbin/mysqld (mysqld 8.0.15-6) starting as process 1
2019-07-04T18:07:07.992788Z 1 [Warning] [MY-012351] [InnoDB] Tablespace 2, name 'sys_operator/status', file './sys_operator/status.ibd' is missing!
2019-07-04T18:07:08.040294Z 0 [System] [MY-010229] [Server] Starting crash recovery...
2019-07-04T18:07:08.052278Z 0 [System] [MY-010232] [Server] Crash recovery finished.
2019-07-04T18:07:08.115053Z 0 [Warning] [MY-010075] [Server] No existing UUID has been found, so we assume that this is the first time that this server has been started. Generating a new UUID: 897f7b0d-9e86-11e9-9245-0e6ff32f5797.
2019-07-04T18:07:08.444061Z 0 [Warning] [MY-010068] [Server] CA certificate ca.pem is self signed.
2019-07-04T18:07:08.563943Z 6 [ERROR] [MY-012592] [InnoDB] Operating system error number 2 in a file operation.
2019-07-04T18:07:08.563966Z 6 [ERROR] [MY-012593] [InnoDB] The error means the system cannot find the path specified.
2019-07-04T18:07:08.563974Z 6 [ERROR] [MY-012216] [InnoDB] Cannot open datafile for read-only: './sys_operator/status.ibd' OS error: 71
2019-07-04T18:07:08.564165Z 6 [ERROR] [MY-000061] [Server] 1812  Tablespace is missing for table `sys_operator`.`status`..
2019-07-04T18:07:08.564399Z 0 [System] [MY-010931] [Server] /usr/sbin/mysqld: ready for connections. Version: '8.0.15-6'  socket: '/var/lib/mysql/mysql.sock'  port: 3306  Percona Server (GPL), Release 6, Revision 63abd08.
2019-07-04T18:07:08.767924Z 0 [System] [MY-011323] [Server] X Plugin ready for connections. Socket: '/var/lib/mysql/mysqlx.sock' bind-address: '::' port: 33060
```

--- a/pkg/sidecar/appconf.go
+++ b/pkg/sidecar/appconf.go
@@ -187,6 +187,13 @@ func initFileQuery(cfg *Config, gtidPurged string) []byte {
 		[]string{"CREATE", "SELECT", "DELETE", "UPDATE", "INSERT"}, fmt.Sprintf("%s.%s", toolsDbName, toolsHeartbeatTableName),
 		[]string{"REPLICATION CLIENT"}, "*.*")...)
 
+	// the slave pod doesn't need to back up sys_operator.status, Xtrabackup might have some bugs that
+	// cause the table to be unclean. We can do this cleanup before the slave pod starts to avoid accidents.
+	if !cfg.IsFirstPodInSet() {
+		queries = append(queries, fmt.Sprintf("DROP TABLE IF EXISTS %s.%s",
+			constants.OperatorDbName, constants.OperatorStatusTableName))
+	}
+
 	// create the status table used by the operator to configure or to mask MySQL node ready
 	// CSV engine for this table can't be used because we use REPLACE statement that requires PRIMARY KEY or
 	// UNIQUE KEY index. Also, the table may exists (in case of pod restart) and should not be changed.

